scripts/patch_websockets.py

```diff
@@ -0,0 +1,101 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Patches uWebsockets Socket.h to fix a crash sending large
+payloads with ssl encoding.
+"""
+from absl import app
+from absl import flags
+
+FLAGS = flags.FLAGS
+
+flags.DEFINE_string("file", None, "Sockets.h file to patch.")
+flags.DEFINE_string("cmakefile", None, "cmake file which downloaded uWebSockets")
+
+# The line of Socket.h that we intend to start overwriting, and the 
+# number of lines to overwite. 
+# See: https://github.com/uNetworking/uWebSockets/blob/e402f022a43d283c4fb7c809332951c3108d6906/src/Socket.h#L309
+REPLACE_BLOCK_START_LINE = 309
```

Comment:
Maybe add a sanity check to ensure that these are the correct lines?

Reply:
Done.